backend/src/main/java/gov/cdc/usds/simplereport/api/model/accountrequest/OrganizationAccountRequest.java

```diff
@@ -0,0 +1,47 @@
+package gov.cdc.usds.simplereport.api.model.accountrequest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import gov.cdc.usds.simplereport.api.model.TemplateVariablesProvider;
+import java.util.HashMap;
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+
+@Getter
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
+public class OrganizationAccountRequest implements TemplateVariablesProvider {
+
+  private static final String TEMPLATE_NAME = "organization-account-request";
+
+  @JsonProperty @NotNull String firstName;
+  @JsonProperty @NotNull String lastName;
+  @JsonProperty @NotNull String email;
+  @JsonProperty @NotNull String workPhoneNumber;
+  @JsonProperty String cellPhoneNumber;
```

Comment:
I think we can remove this cellPhoneNumber field, actually - Nick said it's not currently being collected on the frontend.

@nickclyde, these are the fields the backend is expecting. Does that line up with the frontend form?

Reply:
We are not collecting cellPhoneNumber, correct. We are collecting middle name, but we can skip sending that to the backend, if that's easier.

Reply:
I will remove the `cellPhoneNumber` field

Reply:
Should we add a middle name? @nickclyde @emmastephenson

Reply:
I think for now it's safe to leave off. My understanding is that the middle name field is only going to be included later in the move to Experian, so that users don't have to enter their name on the first page and enter it again on the Experian details page. But for now, the account creation step doesn't need middle name, so I wouldn't include it.